src/std/math.earl

```diff
@@ -42,3 +44,102 @@ module Math
     }
     return x;
```

Comment:
I believe this is a bug from me, but this should be `y`.

Reply:
Tested your functions and they seem to work.